specification/ai/data-plane/Face/preview/v1.1-preview.1/Face.json

```diff
@@ -0,0 +1,7948 @@
+      "required": false,
+      "type": "string",
+      "enum": [
+        "recognition_01",
+        "recognition_02",
+        "recognition_03",
+        "recognition_04"
+      ],
+      "x-ms-enum": {
+        "name": "RecognitionModel",
+        "modelAsString": true,
+        "values": [
+          {
+            "name": "recognition_01",
+            "value": "recognition_01",
+            "description": "The default recognition model for Face - Detect. All those faceIds created before 2019 March are bonded with this recognition model."
+          },
+          {
+            "name": "recognition_02",
+            "value": "recognition_02",
+            "description": "Recognition model released in 2019 March."
+          },
+          {
+            "name": "recognition_03",
+            "value": "recognition_03",
+            "description": "Recognition model released in 2020 May."
+          },
+          {
+            "name": "recognition_04",
+            "value": "recognition_04",
+            "description": "Recognition model released in 2021 February. It's recommended to use this recognition model for better recognition accuracy."
+          }
+        ]
+      },
+      "x-ms-parameter-location": "method"
+    },
+    "FaceDetectionOptions.returnFaceAttributes": {
+      "name": "returnFaceAttributes",
+      "in": "query",
+      "description": "Analyze and return the one or more specified face attributes in the comma-separated string like 'returnFaceAttributes=headPose,glasses'. Supported face attributes include headPose, glasses, occlusion, accessories, blur, exposure, noise, mask, and qualityForRecognition. Face attribute analysis has additional computational and time cost.",
+      "required": false,
+      "type": "string",
+      "x-ms-parameter-location": "method"
+    },
+    "FaceDetectionOptions.returnFaceId": {
+      "name": "returnFaceId",
+      "in": "query",
+      "description": "Return faceIds of the detected faces or not. The default value is true.",
+      "required": false,
+      "type": "boolean",
+      "x-ms-parameter-location": "method"
+    },
+    "FaceDetectionOptions.returnFaceLandmarks": {
+      "name": "returnFaceLandmarks",
+      "in": "query",
+      "description": "Return face landmarks of the detected faces or not. The default value is false.",
+      "required": false,
+      "type": "boolean",
+      "x-ms-parameter-location": "method"
+    },
+    "FaceDetectionOptions.returnRecognitionModel": {
+      "name": "returnRecognitionModel",
+      "in": "query",
+      "description": "Return 'recognitionModel' or not. The default value is false.",
+      "required": false,
+      "type": "boolean",
+      "x-ms-parameter-location": "method"
+    },
+    "ListRequestOptions.start": {
+      "name": "start",
+      "in": "query",
+      "description": "List sessions from the last sessionId greater than the \"start\". It contains no more than 64 characters. Default is empty.",
```

Comment:
This suggests that sessions are returned sorted by sessionId -- is that correct? If so that should be documented.

Reply:
I moved the description to `BaseSessions.listSessions` since ListRequestOptions is shared by other GAed operations, it's not limited to session.